File: Factorial de un número.py
# Leer el valor de N
N = int(input("Ingrese un número entero positivo: "))

# Inicializar el producto
producto = 1

# Calcular el producto de los números desde 1 hasta N
for i in range(1, N + 1):
    producto = producto * i

# Mostrar el resultado
print("El producto desde 1 hasta", N, "es:", producto)

File: Factorial de un número.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    long long producto = 1;

    // Leer el valor de N
    cout << "Ingrese un numero entero positivo: ";
    cin >> N;

    // Calcular el producto de los números desde 1 hasta N
    for (int i = 1; i <= N; i++) {
        producto = producto * i;
    }

    // Mostrar el resultado
    cout << "El producto desde 1 hasta " << N << " es: " << producto << endl;

    return 0;
}
